docs/DeveloperGuide.adoc

```diff
@@ -28,65 +28,78 @@ Refer to the guide <<SettingUp#, here>>.
+link:{repoURL}/blob/master/src/main/java/life/calgo/MainApp.java[`MainApp`].
+This component is responsible for:
+
+* Launching App: Initializes the other components in the correct sequence, and connects them up with each other.
+* Exiting App: Shuts down the components and invokes cleanup method where necessary.
 
-* At app launch: Initializes the components in the correct sequence, and connects them up with each other.
-* At shut down: Shuts down the components and invokes cleanup method where necessary.
+<<<
 
 <<Design-Commons,*`Commons`*>> represents a collection of classes used by multiple other components.
-The following class plays an important role at the architecture level:
+In particular, the `LogsCenter` class plays an important role at the architecture level:
 
-* `LogsCenter` : Used by many classes to write log messages to the App's log file.
+* `LogsCenter` : Writes log messages to the App's log file, for various classes.
 
-The rest of the App consists of four components.
+The rest of the App comprises of four components.
 
-* <<Design-Ui,*`UI`*>>: The UI of the App.
+* <<Design-Ui,*`UI`*>>: The User Interface (UI).
 * <<Design-Logic,*`Logic`*>>: The command executor.
-* <<Design-Model,*`Model`*>>: Holds the data of the App in-memory.
-* <<Design-Storage,*`Storage`*>>: Reads data from, and writes data to, the hard disk.
+* <<Design-Model,*`Model`*>>: The in-memory representation of the App data.
+* <<Design-Storage,*`Storage`*>>: The file manager for reading from and writing to the hard disk.
 
-Each of the four components
+Each of the four components:
 
-* Defines its _API_ in an `interface` with the same name as the Component.
+* Defines its _Application Programming Interface (API)_ in an `interface` with the same name as the Component.
 * Exposes its functionality using a `{Component Name}Manager` class.
 
-For example, the `Logic` component (see the class diagram given below) defines it's API in the `Logic.java` interface and exposes its functionality using the `LogicManager.java` class.
+For example, the `Logic` component (see the class diagram given below) defines its API in the `Logic.java` interface and exposes its functionality using the `LogicManager.java` class.
 
 .Class Diagram of the Logic Component
 image::LogicClassDiagram.png[]
 
+<<<
+
 [discrete]
 ==== How the architecture components interact with each other
 
-The _Sequence Diagram_ below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The _Sequence Diagram_ below shows how the components interact with each other for the scenario where the user issues the command `delete n/Apple`.
 
-.Component interactions for `delete 1` command
+.Component interactions for `delete n/Apple` command
 image::ArchitectureSequenceDiagram.png[]
 
 The sections below give more details of each component.
 
+<<<
+
 [[Design-Ui]]
 === UI component
 
 .Structure of the UI Component
 image::UiClassDiagram.png[]
```

Comment:
The arrow heads pointing to {abstract} UiPart are overlapping. It might be better to combine them?